clar2wasm/src/words/bindings.rs

```diff
@@ -1,8 +1,26 @@
 use clarity::vm::{ClarityName, SymbolicExpression};
+use walrus::{GlobalId, Module};
 
+use crate::error_mapping::ErrorMap;
 use crate::wasm_generator::{ArgumentsExt, GeneratorError, WasmGenerator};
 use crate::words::ComplexWord;
 
+fn get_global(module: &Module, name: &str) -> Result<GlobalId, GeneratorError> {
+    module
+        .globals
+        .iter()
+        .find(|global| {
+            global
+                .name
+                .as_ref()
+                .map_or(false, |other_name| name == other_name)
+        })
+        .map(|global| global.id())
+        .ok_or_else(|| {
+            GeneratorError::InternalError(format!("Expected to find a global named ${name}"))
+        })
+}
```

Comment:
This function is being reused in several parts of the code. Could you please move it to `wasm_utils.rs`? I don't usually prefer this "sweeping functions under the rug" approach, but it’s better than duplicating code.

Reply:
I moved the `get_global` function in `wasm_utils.rs`.

I am now using it by importing the function from `wasm_utils.rs`.